modules/streamer/src/main/StreamerForm.scala

```diff
@@ -27,16 +27,16 @@ object StreamerForm:
         text.verifying("Invalid YouTube channel ID", s => Streamer.YouTube.parseChannelId(s).isDefined)
       ),
       "listed" -> of[Listed],
-      "approval" -> optional(
+      "approval" ->
         mapping(
           "granted"   -> boolean,
           "tier"      -> optional(number(min = 0, max = Streamer.maxTier)),
           "requested" -> boolean,
           "ignored"   -> boolean,
           "chat"      -> boolean,
-          "quick"     -> optional(nonEmptyText)
+          "quick"     -> optional(nonEmptyText),
+          "reason"    -> optional(text)
```

Comment:
so reason can be `Some("")`?

Reply:
I am actually not sure browsers will provide an empty string in a form response when that field started out empty. It's optional in the data structures & form mapping because it's optional in mongodb. but None and Some("") are the same as far as mod logging and streamer notification are concerned.

Reply:
> None and Some("") are the same as far as mod logging and streamer notification are concerned.

And as of https://github.com/lichess-org/lila/pull/16655 this is no longer true.